src/components/plots/barPlot.ts

```diff
@@ -2,6 +2,12 @@
 
 module Plottable {
 export module Plots {
+  export module Bars {
+    export class Mode {
+      public static VERTICAL = "vertical";
+      public static HORIZONTAL = "horizontal";
```

Comment:
how about `Plottable.Orientation.VERTICAL` ? This is not used for plots only. Will be used for Scales as well in the future